main.js

```diff
@@ -0,0 +1,28 @@
+import { selectCharacterQuotes } from "./advice-generator.js";
+
+// Format each character's name with their quote.
+function formatMessage(characterName, quote) {
+  const verbs = ["responds", "answers", "suggests", "advises", "speaks"];
+  const randomIndex = Math.floor(Math.random() * verbs.length);
+  const verb = verbs[randomIndex];
+  return `${characterName} ${verb}: '${quote}' \n`;
+}
+
+function panelResponds(userInput = undefined) {
+  // TO DO: prompt the user for a question
+  console.log(
+    `\nYou approach a panel of legendary figures, their forms obscured.`
+  );
+  console.log(
+    `You speak nothing, yet three figures step forward from the mist.\n`
```

Comment:
Will this always be true if the `userInput` matches on quotes from more than 3 characters?

Reply:
That will definitely need change if there are more than 3 matches -- I'd want all the matches to show. Making this dynamic seems like a good idea